migrations/0001_add_tables.up.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS parcel_status (
+  id serial PRIMARY KEY,
+  status_value TEXT
+);
+
+CREATE TABLE IF NOT EXISTS parcel (
+  id serial PRIMARY KEY,
+  user_id int,
+  carrier_id int,
+  created_at timestamp,
+  updated_at timestamp,
+  status int,
+  source_address TEXT,
+  destination_address TEXT,
+  source_time timestamp,
```

Comment:
what is the use of `source_time`?

Reply:
source time is the time when a carrier will be assigned to carry the parcel.

Reply:
is it required? what will be the benefit of this field?